core/src/main/scala/caliban/Configurator.scala

```diff
@@ -0,0 +1,47 @@
+package caliban
+
+import caliban.CalibanError.ValidationError
+import caliban.execution.QueryExecution
+import caliban.validation.Context
+import caliban.validation.Validator.DefaultValidations
+import zio._
+import zio.prelude.EReader
+
+object Configurator {
+  case class ExecutionConfiguration(
```

Comment:
I think we need to bring the deleted scaladoc from `GraphQL.execute` here to explain the different arguments

Reply:
Good point, done.